test/hotspot/jtreg/compiler/lib/verify/Verify.java

```diff
@@ -71,107 +109,176 @@ private static void checkEQ(Object a, Object b, String context) {
         if (ca != cb) {
```

Comment:
Only seen this in my IDE: `ca` and `cb` should be `Class<?>` instead of the raw `Class` since `getClass()` returns a `Class<?>` (cannot make a suggestion since it's hidden here).

Reply:
fixed